samples/snippets/csharp/VS_Snippets_Wpf/WindowClosingSnippets/CSharp/DataWindow.xaml.cs

```diff
@@ -1,4 +1,3 @@
-//<SnippetWindowClosingCODEBEHIND1>
 using System; // EventArgs
 using System.ComponentModel; // CancelEventArgs
 using System.Windows; // window
```

Comment:
@svick please open an issue for us to look at this. since we have a bunch of legacy code snippets, we'll find all sorts of "treasures" in there 😄